solr/core/src/java/org/apache/solr/cloud/api/collections/OverseerCollectionMessageHandler.java

```diff
@@ -468,36 +467,6 @@ void checkResults(String label, NamedList<Object> results, boolean failureIsFata
     }
   }
 
-
-  //TODO should we not remove in the next release ?
-  private void migrateStateFormat(ClusterState state, ZkNodeProps message, NamedList results) throws Exception {
```

Comment:
Yay!

Reply:
Had to remove 1,660 lines and add 568 to get rid of that comment :)